Scoop/Controller/Onboarding/PhoneNumberViewController.swift

```diff
@@ -29,40 +29,55 @@ class PhoneNumberViewController: OnboardingViewController {
             }
             
             Networking.shared.currentUser.phoneNumber =  phoneNumber
-            
             self.delegate?.didTapNext(navCtrl, nextViewController: nil)
         }
-       
+        
+        setupTitleLines()
+        setBackButtonVisibility(isHidden: true)
```

Comment:
In this case you would just do `backButton.isHidden = true` without needing the helper function